parallel-install/pkg/installation/installation_test.go

```diff
@@ -0,0 +1,93 @@
+package installation
+
+import (
+	"context"
+	"fmt"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/components"
+	"github.com/kyma-incubator/hydroform/parallel-install/pkg/config"
+	"github.com/stretchr/testify/assert"
+	"log"
+	"testing"
+	"time"
+)
+
+const cancelTimeout = 50
+const quitTimeout = 150
+const cancelTimeoutMillisecond = time.Duration(50) * time.Millisecond
+const quitTimeoutMillisecond = time.Duration(150) * time.Millisecond
+
+
+func Test_CancelTimeout(t *testing.T) {
+	//given
+	i := newInstallation()
+
+	ctx, cancelFunc := context.WithCancel(context.TODO())
+
+	hc := &mockHelmClient{
+		cancelWithTimeout: true,
+	}
+	component := newComponent(hc)
+
+	//when
+	err := i.installPrerequisites(ctx, cancelFunc, []components.Component{component}, cancelTimeoutMillisecond, quitTimeoutMillisecond)
+
+	//then
+	assert.Error(t, err, "Kyma installation failed due to the timeout")
```

Comment:
It would be nice here to show that if we have e.g: 5 components and there's the timeout during installation of the third, the fourth and fifth one are skipped. We'd have test coverage for `prerequisites` package "for free" then.